### PR TITLE
send = 0842,0843,0846,08C1,08C2

### PR DESCRIPTION
**Pull request by sctnightcore**

### CODE REVIEW

File: src/Network/Send.pm [hunk 1]
```diff
@@ -1511,4 +1511,42 @@ sub sendchangetitle {
...
+	$self->sendToServer($self->reconstruct({
+		switch => 'remove_aid_sso',
+		ID => $accountID,
+	}));
+}
+
+sub sendmarcostart {
+	my ($self) = @_;
+	$self->sendToServer($self->reconstruct({
+		switch => 'marcostart',
```
Comment (alisonrag) [inline src/Network/Send.pm]: http://openkore.com/index.php/Development_Conventions#Naming

Packet "names": underscore_separated_words

File: src/Network/Send.pm [hunk 2]
```diff
@@ -1511,4 +1511,42 @@ sub sendchangetitle {
...
+	}));
+}
+
+sub sendmarcostop {
```
Comment (alisonrag) [inline src/Network/Send.pm]: did you wanna mean macRo?

File: src/Network/Send.pm [hunk 3]
```diff
@@ -1511,4 +1511,42 @@ sub sendchangetitle {
...
+
+sub sendremoveaidsso {
+	my ($self, $accountID) = @_;
+	$self->sendToServer($self->reconstruct({
+		switch => 'remove_aid_sso',
+		ID => $accountID,
+	}));
+}
+
+sub sendmarcostart {
```
Comment (alisonrag) [inline src/Network/Send.pm]: please follow the convention, use camelCase

File: src/Network/Send.pm (hunk 2 above)
Comment (sctnightcore) [reply src/Network/Send.pm]:
```
	packet(0x08c1,2,clif->pDull); // CZ_MACRO_START
	packet(0x08c2,2,clif->pDull); // CZ_MACRO_STOP
```

oh , I wrong . asdjalskdj

File: src/Network/Send.pm (hunk 1 above)
Comment (sctnightcore) [reply src/Network/Send.pm]: Fix :  46c1338

File: src/Network/Send.pm (hunk 2 above)
Comment (sctnightcore) [reply src/Network/Send.pm]: Fix : 46c1338

File: src/Network/Send.pm (hunk 3 above)
Comment (sctnightcore) [reply src/Network/Send.pm]: Fix : 46c1338

**Pull request merged**